apps/web/src/pages/playground/onboarding/Terminal.tsx

```diff
@@ -15,78 +15,87 @@ interface TerminalComponentProps {
+    fitAddon.current.fit();
+    terminalInstance.current = terminal;
 
-      setTimeout(() => {
-        fitAddon.current.fit();
-      }, 500);
+    setTimeout(() => {
+      fitAddon.current.fit();
+    }, 500);
 
-      const handleResize = () => {
-        fitAddon.current.fit();
-      };
+    const handleResize = () => {
+      fitAddon.current.fit();
+    };
 
-      window.addEventListener('nv-terminal-layout-resize', handleResize);
-      window.addEventListener('resize', handleResize);
+    window.addEventListener('nv-terminal-layout-resize', handleResize);
+    window.addEventListener('resize', handleResize);
 
-      return () => {
-        window.removeEventListener('resize', handleResize);
-        window.removeEventListener('nv-terminal-layout-resize', handleResize);
-        terminal.dispose();
-      };
-    }, []);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+      window.removeEventListener('nv-terminal-layout-resize', handleResize);
+      terminal.dispose();
+    };
+  }, []);
 
-    return (
-      <div className={css({ height: '100%' })}>
-        <div
-          className={css({
-            color: 'typography.text.secondary',
-            bg: '#292933',
-            lineHeight: '20px',
-            fontSize: '14px',
-            p: '4',
-            width: '100%',
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'space-between',
-          })}
-        >
-          <span className={css({ display: 'flex', alignItems: 'center' })}>
-            <IconTerminal className={css({ mr: '4' })} />
-            Terminal
-          </span>
+  return (
+    <div className={css({ height: '100%' })}>
+      <div
+        className={css({
+          color: 'typography.text.secondary',
+          bg: '#292933',
+          lineHeight: '20px',
+          fontSize: '14px',
+          p: '4',
+          width: '100%',
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+          position: 'sticky',
+        })}
+      >
+        <span className={css({ display: 'flex', alignItems: 'center' })}>
+          <IconTerminal className={css({ mr: '4' })} />
+          Terminal
+        </span>
 
-          <Button size="xs" Icon={IconMenuBook} onClick={onStepAddGuide}>
-            How to add a Step?
-          </Button>
-        </div>
-        <div style={{ height: '100%' }} ref={terminalRef} />
+        <Button size="xs" Icon={IconMenuBook} onClick={onStepAddGuide}>
+          How to add a Step?
+        </Button>
       </div>
-    );
-  }
-);
+      <div
+        className={css({
+          height: '100%',
+          // Applying padding to the terminal content.
+          '& .xterm-screen': { padding: '10px' },
+          // !important is necessary to override xterm.js styles.
+          '& .xterm-rows': { fontFamily: 'Menlo, Monaco, "Courier New", monospace !important' },
```

Comment:
```suggestion
          '& .xterm-rows': { fontFamily: 'mono !important' },
```
Use design system value